webpack-widget.js

```diff
@@ -102,6 +231,19 @@ const getLegacyWidgetBuildConfig = (config = {}) => {
 		})
 	}
 
+	let rules = getDefaultJSRules()
+
+	let plugins = {
```

Comment:
This seems half implemented.  There's no point to doing this and then using them in the rules section later unless we want to export them so they can be referenced. 

Either make sure they are exported for use or just define them inline in the rules section below

Reply:
Is this to preserve the current expected default functionality?